components/Screenings.js

```diff
@@ -0,0 +1,30 @@
+import styles from './Screenings.module.css';
+
+const Screenings = ({ screenings }) => {
+  if (screenings.length > 0) {
+    return (
+      <div className={styles.container}>
+        <h4>Screenings</h4>
+        <ul className={styles.screenings}>
+          {screenings.map((screening) => {
+            return (
+              <li className={styles.screening} key={screening.date}>
```

Comment:
If there are two screenings on the same date, that will result in duplicate keys.

It's always best to use a unique ID as the key. If no unique IDs exist in the data, use the index instead (which is passed as the second argument to the callback for `map()`).